storage/innobase/dict/dict0mem.cc

```diff
@@ -1216,24 +1216,24 @@ inline void dict_index_t::reconstruct_fields()
 	n_nullable = 0;
 	ulint n_core_null = 0;
 	const bool comp = dict_table_is_comp(table);
-	const auto* non_pk_col_map = table->instant->non_pk_col_map;
+	const auto* col_map = table->instant->col_map;
```

Comment:
The variable `col_map` refers to something completely different: mapping columns to columns during `ALTER TABLE…ALGORITHM=INPLACE`. I would not use the same name for two different things.

The `dict_instant_t::non_pk_col_map` maps clustered index fields into existing or instantly dropped columns. The high-order bits contain some flags. The first element of the array corresponds to `dict_index_t::first_user_field()`.

I agree that `non_pk_col_map` is not a good name, because indexes do not have columns, but fields (which correspond to column prefixes or full columns). Maybe `user_field_map` or `field_map` would be better, also to document the relationship with `dict_index_t::first_user_field()`.

Reply:
Renamed to `field_map`. I would prefer not to add obvious prefix to name.